rcsb/utils/dictionary/DictMethodEntityHelper.py

```diff
@@ -448,6 +449,12 @@ def buildContainerEntityIds(self, dataContainer, catName, **kwargs):
                     cObj.setValue(",".join(refSeqIdD["dbAccession"]).strip(), "reference_sequence_identifiers_database_accession", ii)
                     cObj.setValue(",".join(refSeqIdD["provSource"]).strip(), "reference_sequence_identifiers_provenance_source", ii)
                     cObj.setValue(",".join(refSeqIdD["dbIsoform"]).strip(), "reference_sequence_identifiers_database_isoform", ii)
+                    uDbName = "UniProt"
+                    uDbIndexL = [i for i, x in enumerate(refSeqIdD["dbName"]) if x == uDbName]
```

Comment:
Not a big problem obviously, but just wondering why the need to set `uDbName = "UniProt"`, if you only reference it once?